primehub-sdk-ct-pipeline/README.md

```diff
@@ -0,0 +1,23 @@
+# PrimeHub SDK CT pipeline - Iris dataset.
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/InfuseAI/showcase/blob/main/primehub-sdk-ct-pipeline/notebook/PrimeHub_SDK_Iris_Model.ipynb)
+
+## Prerequisites
+
+1. PrimeHub EE 3.9.0 version to do the showcase.
+    
+## Usage
+
+If you want to know the step-by-step method, please contact us: support@infuseai.io.
+
+Please run the python file by the following code.
+
+```bash
+python main.py
```

Comment:
Should we need to address what's the minimum Python version? ex. 3.6 or any 3.x?